app/models/student_course.rb

```diff
@@ -9,6 +9,7 @@
 #  self_paced :boolean
 #  created_at :datetime         not null
 #  updated_at :datetime         not null
+#  sent_email :boolean          default(FALSE)
```

Comment:
I'm feeling iffy about this attribute name cause it's not very revealing of why it actually gets flipped. Can you do another migration to change this to `completed`?